packages/sprinkles/README.md

```diff
@@ -209,6 +209,10 @@ document.write(`
 
 ---
 
+⚛️ &nbsp; Using React? Turn your atoms into a `<Box>` component with 🍰 [Dessert Box.](https://github.com/TheMightyPenguin/dessert-box)
```

Comment:
Should the cake be clickable?

```suggestion
⚛️ &nbsp; Using React? Turn your atoms into a `<Box>` component with [🍰 Dessert Box.](https://github.com/TheMightyPenguin/dessert-box)
```

Reply:
It was clickable originally, but the underline is really weird on hover so I moved it out of the link.